pkgs/applications/networking/instant-messengers/dino/default.nix

```diff
@@ -73,6 +73,26 @@ stdenv.mkDerivation rec {
     libxkbcommon
   ];
 
+  cmakeFlags = ["-DBUILD_TESTS=yes"];
+
+  doCheck = true;
+  checkPhase = ''
+  echo "================= Running built tests ================="
+  set -x
```

Comment:
This `set -x`/`set +x` should be the other way around here. On the other hand, why not let a test just fail?

Reply:
Should I use "set -e" here instead then? I based this script on the zlib example in the nixpkgs docs because I'm not super familiar with bash scripting.